frontend/src/components/AccountHistoryInput/index.js

```diff
@@ -6,16 +6,24 @@ import saveDisabled from '@/assets/images/saveDisabled.svg';
 import saveActive from '@/assets/images/saveActive.svg';
 import chevronDown from '@/assets/images/chevronDown.svg';
 import Dropdown from '@/components/Dropdown';
+import controller from '@/controller';
 
 export default class AccountHistoryInput extends Component {
   template() {
-    const { inputIsIncome, isInputDataFilled, inputs } = this.state;
-    const { category, date, content, amount, paymentMethod } = inputs;
+    let { inputIsIncome, isInputDataFilled, inputs } = this.state;
```

Comment:
```suggestion
    let { isInputIncome, isInputDataFilled, inputs } = this.state;
```

짜잘한 것이긴 한데, 이름이 통일이 안되는 느낌이라 요런건 괜찮을 까요?

Reply:
좋습니다. 다른 곳에서도 쓰고있는 변수명이니 고치고 다시 커밋할게요 !